src/web/lib/hakuneko/frontend@classic-dark/mangas.html

```diff
@@ -262,11 +295,41 @@
                 //
             }
 
+            /**
+            * Event listener attached to the refresh button
+            * Checks for new content to read by refreshing the bookmarks, updating the web source and checking the chapters bookmarks
+            */
+            async onCheckNewContent() {
+                if(! this.get('enableCheckNewContent'))
+                {
+                    alert('Please wait 60s before checking again.');
```

Comment:
Clicking on a disabled button should not show any actions

Reply:
I should indeed introduce a simple tooltip instead of an alert